spec/calendar.html

```diff
@@ -0,0 +1,151 @@
+<emu-clause id="sec-temporal-calendar-objects">
+  <h1>Temporal.Calendar Objects</h1>
+  <p>A Temporal.Calendar object is an immutable Object representing a calendar.</p>
+
+  <emu-clause id="sec-temporal-calendar-abstract-ops">
+    <h1>Abstract Operations for Temporal.Calendar Objects</h1>
+
+    <emu-clause id="sec-temporal-getbuiltincalendar" aoid="GetBuiltinCalendar">
+      <h1>GetBuiltinCalendar ( _id_ )</h1>
+      <emu-alg>
+        1. If %Temporal.Calendar%.[[BuiltinCalendars]] doesn't have a key equal to _id_, then
+          1. Throw a *RangeError* exception.
+        1. Let _calendar_ be %Temporal.Calendar%.[[BuiltinCalendars]].[[<_id_>]].
+        1. Return ? Construct(_calendar_, « »).
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal-getdefaultcalendar" aoid="GetDefaultCalendar">
+      <h1>GetDefaultCalendar ( )</h1>
+      <emu-alg>
+        1. Return ? GetBuiltinCalendar(*"iso8601"*).
+      </emu-alg>
+    </emu-clause>
+  </emu-clause>
+
+  <emu-clause id="sec-temporal-calendar-constructor">
+    <h1>The Temporal.Calendar Constructor</h1>
+    <p>The Temporal.Calendar constructor:</p>
+    <ul>
+      <li>is the intrinsic object <dfn>%Temporal.Calendar%</dfn>.</li>
+      <li>creates and initializes a new Temporal.Calendar object when called as a constructor.</li>
+      <li>is designed to be subclassable. It may be used as the value of an `extends` clause of a class definition. Subclass constructors that intend to inherit the specified Temporal.Calendar behaviour must include a super call to the %Temporal.Calendar% constructor to create and initialize subclass instances with the necessary internal slots.</li>
+      <li>has a *"length"* property whose value is 1.</li>
+    </ul>
+
+    <emu-clause id="sec-temporal.calendar">
+      <h1>Temporal.Calendar ( _id_ )</h1>
+      <p>When the `Temporal.Calendar` function is called, the following steps are taken:</p>
+      <emu-alg>
+        1. If NewTarget is *undefined*, then
+          1. Throw a *TypeError* exception.
+        1. Let _calendar_ be ? OrdinaryCreateFromConstructor(NewTarget, *"%Temporal.Calendar.prototype%"*, « [[InitializedTemporalCalendar]], [[ID]] »).
+        1. Set _calendar_.[[ID]] to _id_.
```

Comment:
For TimeZone we use [[Identifier]], I think it should be consistent.

Reply:
Done.